frontend/src/styles/UserList.css

```
@@ -0,0 +1,118 @@
+/* Styles taken from Figma */
+tbody {
+  display: block;
+  height: 511px;
+  overflow-y: scroll;
+}
+
+thead,
+tbody tr {
+  display: table;
+  width: 100%;
+}
+
+.people_table {
+  height: 511px;
+
+  border-radius: 15px;
+  border: 20px solid white;
+  background-color: white;
+  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25);
+  border-spacing: 0;
+}
+
+.people_table_header {
+  height: 51px;
+
+  text-align: start;
+  padding-left: 50px;
+
+  background: rgba(0, 87, 184, 0.1);
+  box-sizing: border-box;
+
+  font-family: Lato;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 22px;
+}
+
+.people_table_header_start {
+  min-width: 200px;
+  height: 51px;
+
+  text-align: start;
+  padding-left: 50px;
+
+  background: rgba(0, 87, 184, 0.1);
+  box-sizing: border-box;
+  border-radius: 11px 0px 0px 11px;
+
+  font-family: Lato;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 22px;
+}
+
+.people_table_header_end {
+  height: 51px;
+
+  text-align: start;
+  padding-left: 50px;
+
+  background: rgba(0, 87, 184, 0.1);
+  box-sizing: border-box;
+  border-radius: 0px 11px 11px 0px;
+
+  font-family: Lato;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 22px;
+}
+
+.people_table_data {
+  min-width: 220px;
+  height: 62px;
+  text-align: center;
+
+  font-family: Lato;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 17px;
+}
+
+.people_table_data_start {
+  min-width: 200px;
+  height: 62px;
+
+  text-align: left;
+  padding-left: 50px;
+  border-radius: 8px 0px 0px 8px;
+
+  font-family: Lato;
+  box-sizing: border-box;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 17px;
+}
+
+.people_table_data_end {
+  height: 62px;
+
+  text-align: center;
+  border-radius: 0px 8px 8px 0px;
+
+  font-family: Lato;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 17px;
+}
+
```

This makes every odd row (i.e. every `2n+1`-th element) have the gray background, instead of having to do it conditionally in JSX:

```suggestion
tr:nth-child(2n) .people_table_data_start,
tr:nth-child(2n) .people_table_data,
tr:nth-child(2n) .people_table_data_end {
  background: rgba(187, 188, 188, 0.2);
}
```